### PR TITLE
Makes the handheld crew pinpointer display who it's pinpointing.

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

Title. Why was this not in the game before I have no idea.

## Why It's Good For The Game

The pinpointer should realistically show who it's tracking.

## Changelog
:cl:
add: handheld pinpointer now shows the name of the person it's tracking
/:cl:

### CODE REVIEW

File: code/game/objects/items/pinpointer.dm [hunk 1]
```diff
@@ -38,6 +38,12 @@
 	toggle_on()
 	user.visible_message("<span class='notice'>[user] [active ? "" : "de"]activates [user.p_their()] pinpointer.</span>", "<span class='notice'>You [active ? "" : "de"]activate your pinpointer.</span>")
 
+/obj/item/pinpointer/examine(mob/user)
```
Comment (Archanial) [inline code/game/objects/items/pinpointer.dm]: Are you sure it's going to work? I already remember something similar runtiming in multitools.
There's also `on_examine()` function.

Comment (ivanmixo) [reply code/game/objects/items/pinpointer.dm]: yeah, tested it, lemme grab a screenshot real quick

Comment (ivanmixo) [reply code/game/objects/items/pinpointer.dm]: image

**Comment by BrumalLake** [comment]
" this dodesnt make sense cause the screeween only showss the arrow on it theres no place to  p ut a name on it"
😭
😭

**Review by francinum — DISMISSED** [review]

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
This is going to get fucky with the subtypes that are made to specifically state what they are tracking.

**Review by Archanial — APPROVED** [review]

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**